src/test/java/uk/gov/hmcts/ccd/data/TestConfiguration.java

```diff
@@ -0,0 +1,34 @@
+package uk.gov.hmcts.ccd.data;
+
```

Comment:
Does it belong to uk.gov.hmcts.ccd.data

Reply:
`TestConfiguration` is used by only one test class at present, `UserProfileRepositoryTest`, though I put it one package level higher - most likely because I was copying the structure from another of our projects :)